crypto/s2n_composite_cipher_aes_sha.c

```diff
@@ -29,17 +29,26 @@
 /* LibreSSL and BoringSSL supports the cipher, but the interface is different from Openssl's. We
  * should define a separate s2n_cipher struct for LibreSSL and BoringSSL.
  */
-#if S2N_OPENSSL_VERSION_AT_LEAST(1,0,1) && !defined(LIBRESSL_VERSION_NUMBER) && !defined(OPENSSL_IS_BORINGSSL)
-#define S2N_AES_SHA_COMPOSITE_AVAILABLE
+#if !defined(LIBRESSL_VERSION_NUMBER) && !defined(OPENSSL_IS_BORINGSSL)
+/* Symbols for AES-SHA1-CBC composite ciphers were added in Openssl 1.0.1:
+ * See https://www.openssl.org/news/cl101.txt.
```

Comment:
This link doesn't work, and I don't see a way to link to the relevant entry in the openssl changelog that does exist. You might want to just delete this line instead of moving it.

Reply:
Well noticed! I'm deleting the link

Reply:
And as I'm already editing the comments, added some hint about the unreliability of composite ciphers on LibreSSL, to make it more future proof.